css/mainPC.css

```
@@ -0,0 +1,155 @@
+@font-face {
+    font-family: "Angst";
+    src: url(/constellations/fonts/angst/static/Angst-Normal.otf);
+}
+
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+
+    width: 100vw;
+    height: 100vh;
+
+    background-image: url('../mainPNG/background.svg');
+    background-repeat: no-repeat;
+    background-size: cover;
+}
+
+.logo {
+    width: 10vw;
+    height: 10vh;
+
+    top: 0;
+}
+
+.main-container {
+    display: flex;
+
+    flex-direction: column;
+    align-items: center;
+
+    height: 90%;
+    width: 95%;
+}
+
+.upper-container {
+    display: flex;
+
+    width: 100%;
+    height: 65%;
+}
+
+.upper-container .left-container {
+    display: flex;
+
+    flex-direction: column;
+
+    width: 11%;
+    height: 100%;
+}
+
+.buttons {
+    display: flex;
+
+    flex-direction: column;
+    justify-content: space-between;
+
+    height: 40%;
+
+    margin-top: 85%;
+}
+
+.button {
+    width: 60%;
+
+    cursor: pointer;
+    user-select: none;
+}
+
+.midle-container {
+    display: flex;
+
+    width: 80%;
+
+    align-items: center;
+    flex-direction: column;
+}
+
+.astro-line {
+    width: 100%;
+    height: 5%;
+
+    border-bottom: solid 2px;
+}
+
+.lower-container {
+    display: flex;
+
+    align-items: center;
+    justify-content: space-between;
+
+    height: 30%;
+    width: 85%;
+}
+
+.block {
+    display: flex;
+
+    justify-content: space-between;
+    align-items: center;
+
+    height: 80%;
+    width: 45%;
+
+    background-color:rgba(255, 255, 255, 0.2);
+    backdrop-filter: blur(1vh);
+
+    padding-top: 1.5%;
+    padding-left: 2%;
+    padding-bottom: 2.5%;
+    padding-right: 2.5%;
+}
+
+.video {
```

сменить имя селектора в соответсвии с конвенцией нейминга, например можно ознакомиться с методологией БЭМ
```suggestion
.home-page__constellation-wheel_video
```